migrations/2023-04-06-092008_create_merchant_ek/up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE MERCHANTKEYSTORE(
+    MERCHANT_ID VARCHAR(255) NOT NULL PRIMARY KEY,
+    KEY BYTEA NOT NULL,
+    CREATED_AT TIMESTAMP NOT NULL DEFAULT now()
+);
```

Comment:
change table name to `merchant_key_store` for consistency with other table names.